.travis.yml

```diff
@@ -26,7 +26,9 @@ env:
         - MAIN_CMD='python setup.py'
         - CONDA_DEPENDENCIES='Cython jinja2'
         - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy h5py matplotlib pyyaml scikit-image pandas pytz beautifulsoup4 ipython mpmath bleach'
-        - PIP_DEPENDENCIES=''
+        # For testing purposes, before these packages are available on pypi
```

Comment:
The changes to this file and to `appveyor.yml` are temporary and should NOT be merged. They are only necessary to support testing against the new plugin packages until they can be released on pypi.